src/main/kotlin/racingcar/Application.kt

```diff
@@ -1,5 +1,82 @@
 package racingcar
 
+import camp.nextstep.edu.missionutils.Console
+import camp.nextstep.edu.missionutils.Randoms
+
 fun main() {
-    // TODO: Implement the program
+
+
+    /**
+     * 1) The user should be able to specify car names and the number of rounds.
+     * - Car names should be comma-separated when entered by the user.
+     * - Each car must have a name, and names cannot exceed 5 characters.
+     */
+
+    println("Enter the names of the cars (comma-separated):")
+    val list = Console.readLine().split(",").map { it.trim() }
+    for (item in list) {
+
+        /**
+         * 6) If the user inputs invalid data, the program should throw an IllegalArgumentException and terminate.
+         */
+        if (item.count() > 5) {
+            throw IllegalArgumentException()
```

Comment:
Including the error message can help with easier debugging and better logging

Reply:
A-ha! It would be nice to write an additional error message!